colorbleed/lib.py

```diff
@@ -214,3 +215,37 @@ def switch_item(container,
     avalon.api.switch(container, representation)
 
     return representation
+
+
+def _get_host_name():
+
+    _host = avalon.api.registered_host()
+    # This covers nested module name like avalon.maya
+    return _host.__name__.rsplit(".", 1)[-1]
+
+
+def collect_container_metadata(container):
+    """Add additional data based on the current host
+
+    If the host application's lib module does not have a function to inject
+    additional data it will return the input container
+
+    Args:
+        container (dict): collection if representation data in host
+
+    Returns:
+        generator
+    """
+
```

Comment:
I think this is fine for now, yet maybe we should add a `# todo` line that states that we should look for simpler/nicer ways to do this?